Scheduler/observing_blocks_auxtel/latiss_daily_calibrations.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "latiss_daily_calibrations",
+    "program": "BIAS_DARK_FLAT",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "auxtel/enable_atcs.py",
+            "standard": true,
+            "parameters": {
+            }
```

Comment:
Could you move this close bracket up to L9? So that it is `"parameters": {}`.

Same for all following empty parameter lines.